webpack.config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
This is a good start, but I want to expand this today to minimize the time our dev builds take